### ISSUE TITLE
HTTP ERROR 500 unable to install

### ISSUE THREAD

**Issue by ciscoterminal** [projectsend/projectsend#658]
## Type

Please complete with an X between the brackets to indicate the type of submission.
This is a:

[] Bug report

[] Feature request

### Description of the problem

Please be as clear as possible

### List the steps to reproduce the issue

1.
2.

### Environment

* ProjectSend version:
* php version:
* MySQL version:

### Additional information

_[Optional]_ Write any additional information that you believe might be useful to reproduce the issue

### Related PR

_[Optional]_ If you have submitted a PR related to this issue, please paste the link here

___

### IMPORTANT

Before submitting an issue, please [do a search on the issues page](https://github.com/ignacionelson/ProjectSend/issues) to make sure that the problem hasn't been reported already.

**Comment by kkplein** [comment]
we are running on debian. we installed an older version first, and then upgraded. We also encountered problems installing the latest master from scratch. Perhaps this upgrade-path will work for you too.

**Comment by ciscoterminal** [comment]
do you mean first install projectsend-legacy then upgrade to new one ?
can you please send me link of that install or zip file 
thanks

**Comment by ignacionelson** [comment]
Oh, this new one is NOT ready to be installed yet. You need to use the legacy one for now.

**Comment by ignacionelson** [comment]
Please always use the latest version that you can get from www.projectsend.org

**Comment by kkplein** [comment]
Yes, I mean install first an older legacy version and then upgrade to latest legacy. Our experience is that the latest legacy version from here does not install well.

**Comment by ignacionelson** [comment]
Yes, I'm working on two bugs (installer and file types restrictions) and plan to upload a test version today

**Comment by ignacionelson** [comment]
Sorry about that! Please try the release candidate for the patched version

https://www.projectsend.org/download/301/

**Comment by ciscoterminal** [comment]
no luck , which os you are using , and what version of PHP and mysql is recommended.

**Comment by ciscoterminal** [comment]
done , 
define('DEBUG', true);

**Comment by ignacionelson** [comment]
So, the config file already exists, and the installer is just blank
http://209.97.177.51/install/

Would it be possible for you to give me sftp access to test it? If you can, please send a user to contact@projectsend.org

If not, a couple of questions. Do you have any error on the php error log? Have you tried setting (just temporarily) the folders permissions to 775? Do you have the pdo extension enabled? Does the database respond to other scripts?

**Comment by ciscoterminal** [comment]
please check your email

**Comment by ignacionelson** [comment]
I didn't get it, can you please re send it? Thank you!

**Comment by kkplein** [comment]
So, we installed latest beta (https://www.projectsend.org/download/301/) and were 'stuck' on the database config screen. (we could only press "check" again and again, not sure why it would not proceed, since everything seemed correct)
So we created sys.config.php manually, and put the correct database infos there. Then going back to the web interface we were taken to the next screen. (site details, admin user, etc)  Then ProjectSend installed nicely.

**Comment by voolik** [comment]
I'm gonna add to what @kkplein said. I just installed the latest version from the homepage and at the installation process, I could not move on from the check page either. Found this error in the log:
```
2019/01/02 12:00:43 [error] 1100#1100: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: <IP address>, server: files.example.com, request: "GET /includes/js/install/make-config.php HTTP/2.0", upstream: "fastcgi://127.0.0.1:9000", host: "files.example.com", referrer: "https://files.example.com/install/make-config.php"
```
There is a GET request made to https://files.example.com/includes/js/install/make-config.php, which returns 404, because there is no install folder in /includes/js/.

**Comment by greecemunky** [comment]
I also experienced the same issue as @kkplein and @voolik .  From what I can tell, after pressing "Check" a call to /includes/js/js.functions.php returns a HTTP 302 Found response, redirecting to "install/make-config.php" which as stated by @voolik does not exist under /includes/js/.   Code for this redirect is found in file sys.vars.php .

I ended up configuring ProjectSend manually by copying /includes/sys.config.sample.php to /includes/sys.config.php then modifying options in /includes/sys.config.php .

**Comment by mirecart** [comment]
I checked the core files laid out on here and found the the news.php referred to in my last post actually is not in your folder so i removed it completely and the site let me in straight away. Very weird

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
